docs/reference/operations/lookup_accounts.md

```diff
@@ -0,0 +1,12 @@
+# `lookup_accounts`
+
+Fetch one or more accounts by their `id`s.
+
+## Event
+
+An [`id`](../accounts.md#id) belonging to a [`Account`](../accounts.md).
+
+## Result
```

Comment:
This is a little confusing in that the result is always a list

Reply:
I think it will make more sense once client implementations handle batching — then from the application's point of view, event/result will be singular, and batching more of an internal implementation detail.